dev.yml

```diff
@@ -16,7 +16,16 @@ commands:
   bootstrap: bin/bootstrap shopify
   console: test/dummy/bin/rails console
   server: test/dummy/bin/rails server -b 192.168.64.1 -p 55330
-  test: bin/rake
```

Comment:
Now that shipit-engine is on Rails 5, you can simply invoke `bin/rails test "$@"`, no need for conditionals.